### PR TITLE
Csv sac

### LINKED ISSUE #2829: Add SAC to the export dive details as csv
### Describe the issue:

- [ ] Bug
- [ ] Change request
- [X ] New feature request
- [ ] Discussion request

### Issue long description:

It would be nice to have the "SAC" field on the list of fields exported on the csv dive details file

### Subsurface version:

4.9.4

Self compiled

80-gce7cc7970540

### PR DESCRIPTION
**Pull request by mturkia**
### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [x] New feature
- [ ] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### Pull request long description:

Adds SAC to be included on dive detail CSV export.

### Related issues:

Fixes #2829

### CODE REVIEW

File: tests/testparse.cpp [hunk 1]
```diff
@@ -378,6 +379,15 @@ void TestParse::exportCSVDiveDetails()
 	clear_dive_file_data();
 
 	parseCSVmanual(1, "testcsvexportmanualimperial.csv");
+
+	// We do not currently support reading SAC, thus faking it
+	if (dive_table.nr > 0) {
+		struct dive *dive = dive_table.dives[dive_table.nr - 1];
+		dive->sac = 1049;
+	}
```
Comment (dirkhh) [inline tests/testparse.cpp:393]: wouldn't `process_loaded_dives()` calculate this for you?

**Review by dirkhh — COMMENTED** [review]
xslt continues to be a weird language :)

not sure about the test - hard coding this isn't terrible, but I wonder if this is the best way of doing it...

File: tests/testparse.cpp (hunk 1 above)
Comment (mturkia) [reply tests/testparse.cpp:393]: Nope, we don't have the samples anymore. But I'll just save the SAC before clearing original data.

**Review by dirkhh — APPROVED** [review]
This looks good. Thanks for jumping on the request!

**Pull request merged**